src/main/java/org/sonarlint/intellij/finding/LiveFinding.java

```diff
@@ -173,7 +167,7 @@ public IssueSeverity getUserSeverity() {
 
   @Override
   public RuleType getType() {
-    return type;
+    return RuleType.SECURITY_HOTSPOT;
```

Comment:
Why do we force `RuleType` to be a `SECURITY_HOTSPOT`? `getType()` is also called for issues